### PR TITLE
Update release tags

### PR DESCRIPTION
**Pull request by pmconne**
Fixes https://github.com/iTwin/itwinjs-backlog/issues/768.

Unaddressed:
- [ ] APIs used strictly in tests - IModelDb.openDgnDb, IModelHost.setHubAccess, BackendHubAccess, BriefcaseLocalValue

Address separately:
- [ ] ClashDetectionServer - not in core, contact package maintainer.
- [ ] AccuDraw - usage of internal APIs appears intended to work around input problems; file separate issue to resolve those underlying problems.
- [ ] getAgendaElements - not in core, contact constraint-solver-frontend package maintainer.

UI and presentation APIs:
- [x] Most (all?) addressed. See details in https://github.com/iTwin/itwinjs-backlog/issues/768. File follow-up issues in appropriate repositories if necessary - not part of itwinjs-core.

Remove usage:
- [x] IModelDb.nativeDb.inlineGeometryParts - transformer can do this for you
- [x] ToolAdmin.fillEventFromCursorLocation - it's already filled at the point where you're calling it
- [x] IModelHost.hubAccess - deleteIModel, createNewIModel

Promoted:
- [x] BilinearPatch
- [x] queryLastModifiedTime
- [x] Entity subclass constructors - made `protected` and `@public` so superclasses in other packages can invoke `super`
- [x] Entity subclass toJSON methods
- [x] Relationship constructor
- [x] Schemas.unregisterSchema
- [x] isSubclassOf
- [x] BriefcaseManager.getBriefcaseBasePath
- [x] EditCommandAdmin runCommand and finishCommand
- [x] IModelApp.toolAdmin.toolSettingsState

Use existing alternative public API:
- [x] ClassRegistry.register - use registerModule
- [x] EditableWorkspaceDb, ITwinWorkspaceContainer, WorkspaceContainer.cloudContainer - use PropertyStore
- [x] IpcApp.callIpcChannel - use makeIpcProxy
- [x] ToolAdmin.onPostInstallTool, onInstallTool, startPrimitiveTool - don't override Tool.run, override individual methods instead. See issue comments for details.
- [x] SchemaContext.getCachedSchemaSync - use getSchemaSync

Use new alternative public API:
- [x] CoreTools - added translatePrompt and translateInput
- [x] StandaloneDb.createClassViews
- [x] IModelHost.hubAccess queryChangeset and queryIModelByName - added same methods to BriefcaseManager
- [x] ChannelAdmin.sharedChannel - added ChannelControl.sharedChannelName
- [x] ToolAdmin.sendButtonEvent - use AccuDraw.sendDataPoint

### CODE REVIEW

File: core/backend/src/IModelDb.ts [hunk 1]
```diff
@@ -1366,8 +1366,9 @@ export namespace IModelDb { // eslint-disable-line no-redeclare
       return this.tryGetModelJson({ id });
     }
 
-    /** Query for the last modified time of the specified Model.
-     * @internal
```
Comment (kabentley) [inline core/backend/src/IModelDb.ts:1370]: Before we make methods like this @public, should we make them take named arguments?

missing “if” in comment.

File: core/frontend/src/tools/ToolAssistance.ts [hunk 2]
```diff
@@ -64,6 +64,48 @@ export enum ToolAssistanceInputMethod {
   Touch,
 }
 
+/** Common Tool Assistance main instruction prompt messages
+ * @public
+ * @extensions
+ */
+export enum ToolAssistancePromptKey {
```
Comment (kabentley) [inline core/frontend/src/tools/ToolAssistance.ts:71]: I prefer to use const members of an “as const” object over string enums. It’s more efficient unless you plan to do reverse lookups.

File: core/backend/src/IModelDb.ts (hunk 1 above)
Comment (pmconne) [reply core/backend/src/IModelDb.ts:1370]: Added "if".
I don't really want to encourage proliferation/expansion of these convenience APIs. If people want to execute a more complicated query for last modified time than "by id", they can write ECSql to do it.

File: core/frontend/src/tools/ToolAssistance.ts (hunk 2 above)
Comment (pmconne) [reply core/frontend/src/tools/ToolAssistance.ts:71]: I'll switch it around.

File: core/backend/src/IModelDb.ts (hunk 1 above)
Comment (kabentley) [reply core/backend/src/IModelDb.ts:1370]: Yeah, that’s probably why it was @internal to begin with. But you only get one chance to set something in stone..

File: core/frontend/src/tools/ToolAssistance.ts (hunk 2 above)
Comment (pmconne) [reply core/frontend/src/tools/ToolAssistance.ts:71]: Changed.

**Review by kabentley — APPROVED** [review]

**Review by bbastings — APPROVED** [review]

**Review by aruniverse — APPROVED** [review]

**Review by wgoehrig — APPROVED** [review]

**Pull request merged**